src/adapter/cloudflare-pages/handler.ts

```diff
@@ -0,0 +1,13 @@
+// @denoify-ignore
+import type { EventContext } from '@cloudflare/workers-types'
+import type { Hono } from '../../hono'
+import type { Env } from '../../types'
+
+export const handle = <E extends Env>(app: Hono<E>) => {
+  return (eventContext: EventContext<{}, string, {}>) => {
```

Comment:
Shouldn't the type annotation for `eventContext` be `EventContext<E["Bindings"], string, {}>`? I'm not familiar with what `Env["Variables"]` is meant for, so `E["Bindings"]` is not the correct type argument either. Don't care about this suggestion if I'm wrong.

Reply:
Thank you for the comment! I think `<{}, string, {}>` should be kept. This is OK because `Env` in Pages and `E['Bindings']`are different things, and users do not touch this part.

Reply:
OK, then let's keep that part as is!